server/api/api.graphql

```diff
@@ -0,0 +1,8 @@
+type Query {
+	me: User

```

Comment:
why are there two `api/api.graphql`s? Forget to delete one?

Reply:
It's cruft.  I'll will make sure to delete

Reply:
I couldn't find this locally; otherwise I would have removed it

Reply:
hmm, can you check the gitignore's and try force pushing to remove this? I see it on my end on a fresh pull of your branch